### DISCUSSION TITLE
Reduce game text size like (for example) character chat and item text for 'Fixed-Classic Layout'

### DISCUSSION THREAD

**Discussion by discipleofkorrin** [runelite/runelite#15835] [Ideas]
[Like this.](https://i.redd.it/3yx6k3t3guq91.png)  I'd love to use this layout but the text size being too big forces me to play in resizable mode instead.  Loads of people use Fixed-Classic Layout so I think this would be a huge addition to RuneLite.

**Comment by IromTypo** [comment]
I'd like this as well, but the other way around. My chatbox text is too small to read without bending over.
